docker-compose.yml

```diff
@@ -34,7 +34,7 @@ consul:
   dns:
     - 127.0.0.1
   labels:
-    - triton.cns.services=consul
+    - triton.cns.services=wp-consul
```

Comment:
This is a breaking change for upgrades from previous versions of this project, but we've discovered that namespacing Consul is critical for supporting multiple projects in a single DC